config/initializers/session_store.rb

```diff
@@ -1,3 +1,3 @@
 # Be sure to restart your server when you modify this file.
 
-Rails.application.config.session_store :cookie_store, key: '_cmr-metadata-review_session'
+Rails.application.config.session_store :active_record_store, :key => '_cmr-metadata-review_session'
```

Comment:
you should keep the syntax as `key:` (they're equivalent but this one is preferred if it is possible)